.travis.yml

```diff
@@ -85,6 +80,7 @@ install:
 script:
   - if [[ $TEST_COVERAGE == 'true' ]]; then composer test-coverage ; else composer test ; fi
   - if [[ $CS_CHECK == 'true' ]]; then composer cs-check ; fi
+  - docker-compose run --rm php
```

Comment:
This is duplicated, it's the 2nd time tests are executed

Reply:
Correct, we want to run tests in Travis and run them again in Docker.